package.json

```diff
@@ -34,10 +34,15 @@
   "dependencies": {
     "@babel/core": "^7.4.5",
     "@babel/polyfill": "^7.4.4",
+    "eslint-plugin-react": "^7.13.0",
```

Comment:
@Ogutu-Brian The package` eslint-plugin-react` is included as both a dev and production dependency. Is that okay?

Reply:
line 32 and 37

Reply:
Hello @Bakley ,
It should be a dev dependency. Let me fix that